src/PrestaShopBundle/Entity/AdminFilter.php

```diff
@@ -210,6 +217,26 @@ public function getFilter()
         return $this->filter;
     }
 
+    /**
+     * @return string
+     */
+    public function getClassName()
```

Comment:
is it really expected to be a class name, or an unique key?

Reply:
in theory it can be anything, but intention is to use class name, because i didnt want to introduce new method for `Filters` like `getUniqueKey()` or something. 

but regarding naming, maybe it could be `key` or similar. :thinking:

Reply:
What if we have 2 grids used in 2 actions from the same class controller?

Which class name is expected here?

Reply:
ummm, what do you mean? can you given code example?